Jenkinsfile

```diff
@@ -0,0 +1,33 @@
+pipeline {
+    agent {
+      docker {
+        image 'python:3.8'
+        args '-u root --cap-add NET_ADMIN'
+      }
+    }
+
+    environment {
+      DISABLE_AUTH = 'true'
+      DB_ENGINE = 'sqlite'
+    }
+
+    stages {
+      stage('Build') {
+        steps {
+          sh 'python3 --version'
+        }
+      }
+
+    stage('Testing') {
+      steps {
+        sh 'pip3 install django'
```

Comment:
Lets move the installation of these dependencies to a requirements.txt file like so:

1 ) Install the required dependencies then do a `pip freeze > requirements.txt` 
2) In `Jenkinsfile` do a `pip install -r requirements.txt`

Eventually we want to move to using Make to abstract these provisioning steps away so we can reuse them outside of the CI environment.